src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/WindowsSettings.json

```diff
@@ -0,0 +1,1748 @@
+    "Type": "SettingsApp",
+    "Command": "ms-settings:startupapps"
+  },
+  {
+    "Name": "VideoPlayback",
+    "Area": "Apps",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:videoplayback"
+  },
+  {
+    "Name": "Notifications",
+    "Area": "Cortana",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:cortana-notifications"
+  },
+  {
+    "Name": "MoreDetails",
+    "Area": "Cortana",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:cortana-moredetails"
+  },
+  {
+    "Name": "PermissionsAndHistory",
+    "Area": "Cortana",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:cortana-permissions"
+  },
+  {
+    "Name": "WindowsSearch",
+    "Area": "Cortana",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:cortana-windowssearch"
+  },
+  {
+    "Name": "CortanaLanguage",
+    "Area": "Cortana",
+    "Type": "SettingsApp",
+    "AltNames": [ "Talk" ],
+    "Command": "ms-settings:cortana-language"
+  },
+  {
+    "Name": "Cortana",
+    "Area": "Cortana",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:cortana"
+  },
+  {
+    "Name": "TalkToCortana",
+    "Area": "Cortana",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:cortana-talktocortana"
+  },
+  {
+    "Name": "AutoPlay",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:autoplay"
+  },
+  {
+    "Name": "Bluetooth",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:bluetooth"
+  },
+  {
+    "Name": "ConnectedDevices",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:connecteddevices"
+  },
+  {
+    "Name": "DefaultCamera",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "DeprecatedInVersion": 1809,
+    "Note": "NoteDeprecated1809",
+    "Command": "ms-settings:camera"
+  },
+  {
+    "Name": "MouseAndTouchpad",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Note": "NoteTouchpad",
+    "Command": "ms-settings:mousetouchpad"
+  },
+  {
+    "Name": "PenAndWindowsInk",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:pen"
+  },
+  {
+    "Name": "PrintersAndScanners",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:printers"
+  },
+  {
+    "Name": "Touchpad",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Note": "NoteTouchpad",
+    "Command": "ms-settings:devices-touchpad"
+  },
+  {
+    "Name": "Typing",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:typing"
+  },
+  {
+    "Name": "Usb",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Command": "ms-settings:usb"
+  },
+  {
+    "Name": "Wheel",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "Note": "NoteDialPaired",
+    "Command": "ms-settings:wheel"
+  },
+  {
+    "Name": "Phone",
+    "Area": "Devices",
+    "Type": "SettingsApp",
+    "AltNames": [ "MobileDevices" ],
+    "Command": "ms-settings:mobile-devices"
+  },
```

Comment:
```suggestion
  {
    "Name": "Phone",
    "Area": "Phone",
    "Type": "SettingsApp",
    "AltNames": [ "MobileDevices" ],
    "Command": "ms-settings:mobile-devices"
  },
```

It seems to be the same as `add your phone`:
![image](https://user-images.githubusercontent.com/61519853/117570074-232d0480-b0c9-11eb-8384-c7c181ef6fad.png)

Reply:
you are right :-)